x-pack/plugins/alerting/server/application/maintenance_window/methods/create/create_maintenance_window.ts

```diff
@@ -46,7 +46,13 @@ export async function createMaintenanceWindow(
           undefined,
           [{ query: scopedQuery.kql, language: 'kuery' }],
           scopedQuery.filters as Filter[],
-          esQueryConfig
+          {
```

Comment:
Could we update `create_maintenance_window.test.ts` to ensure these are now passed?